1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Unit tests
+subtitle: Developer guide
+---
+
+<link rel="import" href="../../../0.5/components/google-youtube/google-youtube.html">
+
+{% include toc.html %}
+
+Use Web Component Tester (`wct`) to unit test your Polymer elements. 
+
+`wct` is an end-to-end testing environment built by the Polymer core team. It
+enables you to test your elements locally, against all of your installed
+browsers, or remotely, via Sauce Labs. `wct` is composed of the following
+tools:
+
+*   [Mocha](https://mochajs.org/) for test framework, complete with support 
+    for BDD and TDD.
+*   [Chai](http://chaijs.com/) for assertions. 
+*   [Sinon](http://sinonjs.org/) for spies, stubs, and mocks.
+*   [Selenium](http://www.seleniumhq.org/) for running tests against 
+    multiple browsers. 
+
+## Quick start
+
+Watch the Polycast below to learn how to set up `wct` and create and run
+some simple unit tests.
+
+<div class="yt-embed">
+  <google-youtube
+    videoid="YBNBr9ECXLo"
+    autoplay="0"
+    rel="0"
+    fluid>
+  </google-youtube>
+</div>
+
+If you prefer step-by-step instructions, the instructions below 
+teach you how to set up `wct`, create a unit test for
+[`<seed-element>`](https://github.com/PolymerElements/seed-element), and then
+run the test. Note that `<seed-element>` is already set up to support `wct`,
+this is just an example of how to set up your own elements to support `wct`.
+
+1.  Install Web Component Tester globally so that you can run it from 
+    the command line.
+
+        npm -g install web-component-tester
+
+    On Mac OS X you need to [manually install][selenium] the latest Safari 
+    extension for Selenium. See the 
+    [Web Component Tester Polycast][workaround-example] for 
+    a demonstration.
+
+1.  `cd` to the base directory of the element.
+
+1.  (Optional) Install and save `wct` locally as a bower component so that your 
+    tests can always import the `wct` runtime.
+
+        bower install --save Polymer/web-component-tester
+
+    The global `wct` that you installed earlier actually serves its own 
+    copy of the `wct` runtime (`browser.js`) whenever it encounters any URL
+    that ends with `web-component-tester/browser.js`. Installing it locally
+    is just a precaution. 
+
+1.  Create a directory for the tests. 
+
+        mkdir test
+
+    When you run `wct` with no arguments, it automatically searches for
+    a directory named `test` and runs any tests it finds in there. You
```

Comment:
"You can use" sounds more like permission... "If you use ..., you'll need to specify..." sounds a little more like a warning. In this case, I'd prefer the second because standard naming == good.